rust/crd/src/lib.rs

```diff
@@ -295,16 +296,23 @@ pub struct SupersetConfig {
     /// Row limit when requesting chart data.
     /// Corresponds to ROW_LIMIT
     pub row_limit: Option<i32>,
+
     /// Maximum number of seconds a Superset request can take before timing out.
     /// This setting affects the maximum duration a query to an underlying datasource can take.
     /// If you get timeout errors before your query returns the result you may need to increase this timeout.
     /// Corresponds to SUPERSET_WEBSERVER_TIMEOUT
-    pub webserver_timeout: Option<u32>,
+    #[fragment_attrs(schemars(
+        description = "Maximum number of seconds a Superset request can take before timing out. See https://docs.stackable.tech/home/nightly/concepts/duration for more information on the duration format"
```

Comment:
I plan to have a look at the fragment generation regarding the descriptions today (see https://github.com/stackabletech/hdfs-operator/pull/407#discussion_r1356529851)

Anyway, we should rename to something like "Maximum time period a Superset request can take before timing out"

Reply:
Yes, ideally we would use the doc comments for the description. For now, I adjusted the attribute as suggested.

Reply:
Fixed in 83fd303